examples/run_emodel/run_emodel.py

```diff
@@ -52,26 +52,17 @@
 )
 ######################################################
 
-def getHoldingThreshCurrent(directory_path):
-    pattern = os.path.join(directory_path, 'EM_*' + 'json')
-    final = glob.glob(pattern)
-    if final:
-        file_name = final[0]
-        with open(file_name, 'r') as file:
-            data = json.load(file)
-    else:
-        raise FileNotFoundError(f"No EModel resource found in {directory_path}.")
-
-    holding_current = 0
-    threshold_current = 0
-
-    for feature in data['features']:
-        if 'soma.v.bpo_holding_current' in feature['name']:
-            holding_current = feature['value']
-            print(feature)
-        elif 'soma.v.bpo_threshold_current' in feature['name']:
-            threshold_current = feature['value']
-            print(feature)
+def getHoldingThreshCurrent(forge, res_em):
```

Comment:
maybe set somewhere default values (None) for holding_current and threshold_current in case we can't find the EModelScript, or if we find it but holding and threshold are not set there